ptx/S_12_4_FTCLI.ptx

```diff
@@ -23,47 +21,33 @@
     </objectives>
 
     <p>
-      In some of the activities and examples in this chapter we have
-      encountered situations where <m>C_1</m> and <m>C_2</m> are different oriented curves from a
-      point <m>P</m> to a point <m>Q</m> and <m>\int_{C_1}\vF\cdot d\vr =
-      \int_{C_2}\vF\cdot d\vr</m>.
-      In this section, we explore vector fields
-      which have the property that for all points <m>P</m> and <m>Q</m>,
-      if <m>C_1</m> and
-      <m>C_2</m> are oriented paths from <m>P</m> to <m>Q</m>,
-      then <m>\int_{C_1}\vF\cdot
+      In some of the activities and examples in this chapter we have encountered situations where <m>C_1</m> and <m>C_2</m> are different oriented curves from a
+      point <m>P</m> to a point <m>Q</m> and <m>\int_{C_1}\vF\cdot d\vr = \int_{C_2}\vF\cdot d\vr</m>. In this section, we explore vector fields which have the property that for all points <m>P</m> and <m>Q</m>, if <m>C_1</m> and <m>C_2</m> are oriented paths from <m>P</m> to <m>Q</m>, then <m>\int_{C_1}\vF\cdot
       d\vr = \int_{C_2}\vF\cdot d\vr</m>.
     </p>
 
     <exploration xml:id="PA_12_4">
       <p>
-        In <xref ref="A_12_3_2">Activity</xref>
-        we considered the vector field
-        <m>\vF(x,y) =
-        \langle y^2,2xy+3\rangle</m> and two different oriented curves from
-        <m>(1,0)</m> to <m>(-1,0)</m>.
-        We found that the value of the line integral of
-        <m>\vF</m> was the same along those two oriented curves.
-
-        <ol>
-          <li>
+        In <xref ref="A_12_3_2">Activity</xref> we considered the vector field <m>\vF(x,y) = \langle y^2,2xy+3\rangle</m> and two different oriented curves from
```

Comment:
The text between the `exploration` or `activity` tag and the first `task` tag needs to be wrapped in `introduction` tags. Any text within an `exploration` or `activity` after the last `task` needs to be inside a conclusion.